app/src/components/Market/OptionsTable/OptionsTable.tsx

```diff
@@ -72,7 +72,14 @@ const OptionsTable: React.FC<OptionsTableProps> = (props) => {
             {options[type].map((option, i) => {
               const { breakEven, price, strike, address } = option
               return (
-                <TableRow key={address}>
+                <TableRow
+                  key={address}
```

Comment:
In my branches order context, the addItem function includes an orderType string with more options than the boolean 'buy or mint'.  Therefore, clicking the row should open a trade order by using the syntax  'onAddItem(option, 'BUY')'

Reply:
currently there are BUY SELL MINT EXEC LP and WLP orderTypes

Reply:
okay, ill make a change with that exact syntax? 'onAddItem(option, 'BUY')'

Reply:
If you want to start the user at the BUY order screen, that is correct

Reply:
Ill add a enum later